### ISSUE TITLE
Couldn't print landmarks

### ISSUE THREAD

**Issue by ProudCule** [ageitgey/face_recognition#585]
* face_recognition version: 1.2.2
* Python version:3.6
* Operating System:MacOS

### Description

I am trying to print out landmarks using both 68 point model and 5 point model

### What I Did

```
face_recognition.face_landmarks(frame)
face_recognition.face_landmarks(frame, face_locations=None, model='small')

the default large model works, however if i try to use it for small 5 point one , it gives out error
and the error i get is 

TypeError: face_landmarks() got an unexpected keyword argument 'model'

```

**Comment by ageitgey** [comment]
I think I need to release that update as a new version. That change isn't released yet.

**Comment by ageitgey** [comment]
Fixing in #596

**Comment by ageitgey** [comment]
Released as v1.2.3
